auth/auth.js

```diff
@@ -1,24 +1,12 @@
-let jwt = require("jsonwebtoken");
-
-const { SECRET } = require("../config/config");
 const utils = require("../utils/index");
 
 exports.testAuth = async (req, res, next) => {
   try {
     let token = await utils.token.extractToken(req);
 
     if (token) {
-      jwt.verify(token, SECRET, (err, decoded) => {
-        if (err) {
-          res.send({
-            success: false,
-            message: "Token is not valid"
-          });
-        } else {
-          res.send({ decoded });
-          next();
-        }
-      });
+      let result = await utils.token.verifyToken(token);
+      res.send(result.decoded);
```

Comment:
what if in case of error if jwt id not decoded then result.decoded will be undefined